sdk/lib/drivers/ip/transport/tcp/tcp.c

```diff
@@ -379,10 +379,10 @@ NTSTATUS TCPConnect
         }
 
         /* Allocate the port in the port bitmap */
-        Connection->AddressFile->Port = TCPAllocatePort(LocalAddress.Address[0].Address[0].sin_port);
-
+        UINT nPort = TCPAllocatePort(LocalAddress.Address[0].Address[0].sin_port);
```

Comment:
Please move the `UINT nPort;` at the top of the `if (!Connection->...)` block.

Reply:
OK, will be done in next commit

Reply:
Resolved.